packages/react/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ExpressiveModal } from '../ExpressiveModal';
+import { ModalBody } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { DDS_LIGHTBOX } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Lightbox Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title LightBox media title
+ * @param {string} props.copy Lightbox media short description
+ * @param {string} props.image Lightbox responsive image object
+ * @param {boolean} props.open sets whether the modal is open/close
+ * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @returns {*} JSX Object
+ */
+const Lightbox = ({ title, copy, image, open, onClose }) => {
```

Comment:
I know it's a little longer, but the name `Lightbox` itself doesn't really mean much here. Technically the `ExpressiveModal` is a "lightbox" already. I think this should be called `LightboxMediaViewer`

Reply:
I would actually like @wonilsuhibm 's opinion on this as well. Do we envision having this component be specific to being a media viewer, or do we want to leave it open for other things in the future?

Reply:
We can totally pull in @wonilsuhibm to get his input, but `LightboxMediaViewer` is what he and I aligned on. Images for now, possibly video, and audio (not sure about pdfs and such, maybe?) in the future. @jeffchew Is there something other than those you're thinking of could go in here? Anything else would most likely just use the `ExpressiveModal`.

Reply:
it sounds like we are good to go with the name LightboxMediaViewer.
@wonilsuhibm @jeoffw @photodow 
I am cool with new name too. Should we be moving forward ?

Reply:
@sannminwin let's go with it. And I think you tagged the wrong Jeff.

Reply:
updated